packages/@jsii/integ-test/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "composite": false,
```

Comment:
Why? I reckon this also disables `incremental`...

Reply:
tsc throws `tsconfig.json(3,5): error TS6304: Composite projects may not disable declaration emit.` ?

Reply:
Oh yeah, right. Composite requires declarations... but why bother?

Reply:
yeah idk, I wasn't quite sure the ideal solve here. Outputting declarations is fine? Just doesn't feel like we should need to do.